src/harness/harness.ts

```diff
@@ -1642,7 +1647,8 @@ module Harness {
         }
 
         function writeComparison(expected: string, actual: string, relativeFilename: string, actualFilename: string, descriptionForDescribe: string) {
-            if (expected != actual) {
+            var encoded_actual = (new Buffer(actual)).toString('utf8')

```

Comment:
This Buffer is the native node buffer. Though browserify version will be nicer as we can run in browser too. There will be additional installation to use it so I will let everyone know once checkin